lease/lessor_test.go

```diff
@@ -45,10 +45,16 @@ func TestLessorGrant(t *testing.T) {
 	defer le.Stop()
 	le.Promote(0)
 
-	l, err := le.Grant(1, 1)
+	ttl := int64(1) // ttl <= minLeaseTTL in this test case
```

Comment:
please don't introduce this new variable, there's more changes from adding `ttl` than code for what this patch is testing